frame/pablo/src/lib.rs

```diff
@@ -685,6 +685,9 @@ pub mod pallet {
 				end_block,
 				reward_configs,
 				lock,
+				share_asset_id: primitives::currency::CurrencyId::xPBLO.0.into(),
+				// TODO: Derive default fNFT collection ID for staking
+				financial_nft_asset_id: todo!(),
```

Comment:
1. We can create a new range for fNFT asset IDs as well
2. We can use a special one in the range for the PBLO and PICA stake fNFTs

Reply:
4a92e26f93c539223ab07f4cc29e85a06a6ef284